amundsen_application/static/js/components/TableDetail/DetailList/DetailListItem/index.tsx

```diff
@@ -72,10 +72,10 @@ class DetailListItem extends React.Component<DetailListItemProps, DetailListItem
     return (
       <li className='list-group-item detail-list-item'>
         <div className={'column-info ' + (isExpandable ? 'expandable' : '')} onClick={ isExpandable? this.onClick : null }>
-          <div style={{ width: '100%', display: 'inline-grid' }}>
+          <div style={{ width: '100%', overflowWrap: 'break-word' }}>
```

Comment:
Should this be a class instead of inline styles?

Reply:
Yeah let's change that.